apps/examples/secure_storage/Kconfig

```diff
@@ -0,0 +1,15 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_SSTORAGE
+	bool "Secure Storage example"
+	default n
+	depends on ARMV8M_TRUSTZONE
```

Comment:
Since you said it is only AMEBAD specific example, may be you can add AMEBAD CONFIG dependency here ?

Reply:
Added CONFIG_SE_AMEBA dependency.